keyboards/handwired/ferris/info.json

```diff
@@ -0,0 +1,46 @@
+{
+    "keyboard_name": "ferris",
+    "url": "https://github.com/pierrechevalier83/ferris/",
+    "maintainer": "@pierrec83",
+    "width": 12,
+    "height": 7.8,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+				{"x":2, "y":0},
+				{"x":9, "y":0},
+				{"x":3, "y":0.28},
+				{"x":8, "y":0.28},
+				{"x":1, "y":0.31000000000000005},
```

Comment:
Please regenerate this without the rotation on the keys - the info.json format, and thus the converter, doesn't support it.